streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -185,7 +213,8 @@ public int compare(final TopicPartition p1,
 
     private TaskManager taskManager;
     private PartitionGrouper partitionGrouper;
-    private AtomicBoolean versionProbingFlag;
+    // error code for the streams app, replaces the original versionProbingFlag
```

Comment:
comment can be omitted

Reply:
I want to mention versionProbingFlag just in case some developer who knew the flag comes wondering what happened to the flag :-)

Reply:
`git blame` is there friend :) -- that's why there is a commit history. Allows us to keep the code base clean :)